app/assets/javascripts/admin/accepted_payments.js

```diff
@@ -1,5 +1,3 @@
 $(document).ready(function() {
-  $('#service_accepted_payments').select2({
-    placeholder: 'Select one or more payment methods'
-  });
+  $('#service_accepted_payments').select2();
```

Comment:
I think it could make sense to replace all of these JS files with a single `.select2` class on these selects. Then the shared code could just be:

```javascript
$(document).ready(function() { $('.select2').select2() })
```